src/main/java/io/polivakha/mojo/properties/ReadPropertiesMojo.java

```diff
@@ -195,6 +198,9 @@ private void loadProperties( Resource resource ) throws MojoExecutionException {
                     project.getProperties().load( stream );
                 }
             }
+
+            duplicatePropertyVerifier.verifyExistedProperties( oldProperties, project.getProperties(), getLog() );
+
```

Comment:
Are you sure this will work?

`project.getProperties()` always return the same properties, so when we do 

```
projectProperties.put( keyPrefix + key, properties.get( key ) );
```

we populate `oldProeprties` as well, because `oldProperties` is just another reference to the same object. This will defeat the purpose of the PR

Reply:
I did copy of properties before loading, using new HashSet :)
Tested on method io.polivakha.mojo.properties.ReadPropertiesMojoTest#givenTwoPropertiesFileWithTwoPatterns_whenResolvingExistingProperties_thenLastPatternFilesTakePrecedence

Reply:
@Mikhail2048, thanks for simplest solution. Pushed your solution. Please, review